app/assets/stylesheets/pdf/_summary.scss

```diff
@@ -1,43 +1,187 @@
-body {font: 12px/1.231 Arial,Helvetica,sans-serif; padding-top: 100px;}
+body {
+  font: 12px/1.231 Arial, Helvetica, sans-serif;
+  padding-top: 100px;
+}
 
 #xyz_main {
-    position: absolute;
-    top: 0px;
-    left: 50%;
-    height: 200px !important;
-    width: 100px;
-    margin-left: -55px;
-    text-align: left;
-}
-
-#quadicon {color: #e3e4e4;float:left; height: 72px; padding: 0px 1px 1px 0px;position: relative;text-transform: none;width: 72px;}
-#quadicon img {border:0px;}
-
-.flobj { position: absolute;z-index: auto;width: 72px;}
-.flobj p { margin: 0 0 0 0;padding: 0;color: #e3e4e4;vertical-align: middle;text-align:center;text-transform: none;text-shadow: 0 0 3px #000;font: normal 16px OpenSansSemibold,Arial,Helvetica,sans-serif !important;}
-.a72 { padding: 3px  0 0 5px; width: 28px; height: 28px;font-size: 1.3em;line-height: 1.8em;}
-.b72 { padding: 3px 0 0 37px; width: 28px; height: 28px;font-size: 1.3em;line-height: 1.8em;}
-.c72 { padding: 38px 0 0 5px; width: 31px; height: 31px;font-size: 1.3em;line-height: 1.7em;}
-.d72 { padding: 38px 0 0 39px; width: 31px; height: 31px;font-size: 1.3em;line-height: 1.7em;}
-.e72 { padding: 6px  0 0 6px; width: 35px; height: 35px;font-size: 1.3em;line-height: 2em;}
```

Comment:
The quadrant classes are much easier to follow with each on a single line. I'd say this should be the one exception to the rule.